EssentialCSharp.Web.Tests/RouteConfigurationServiceTests.cs

```diff
@@ -0,0 +1,65 @@
+using EssentialCSharp.Web.Services;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace EssentialCSharp.Web.Tests;
+
+public class RouteConfigurationServiceTests : IClassFixture<WebApplicationFactory<Program>>
+{
+    private readonly WebApplicationFactory<Program> _Factory;
+    private readonly IRouteConfigurationService _RouteConfigurationService;
+
+    public RouteConfigurationServiceTests(WebApplicationFactory<Program> factory)
+    {
+        _Factory = factory;
+
+        // Get the service from the DI container to test with real routes
+        var scope = _Factory.Services.CreateScope();
```

Comment:
Scopes are disposable, I think this is something that we would want cleaned up.

One option would be to do the scope creation/cleanup, inside of a derived class from `WebApplicationFactory<Program>`